src/Tribe/Views/V2/Assets.php

```diff
@@ -634,4 +650,19 @@ public function should_enqueue_single_event_block_editor_styles() {
 
 		return true;
 	}
+
+	/**
+	 * Load assets on the add or edit pages of the block editor only.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool
+	 */
+	public function should_enqueue_admin() {
+		if ( is_admin() && get_current_screen()->is_block_editor ) {
+			return true;
+		}
```

Comment:
Split these into two separate inverted conditionals returning false and return true on the end.

Reply:
@bordoni done.